WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+QuickStart.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+private var observer: NSObjectProtocol?
+
+extension MySiteViewController {
+
+    func startObservingQuickStart() {
+        observer = NotificationCenter.default.addObserver(forName: .QuickStartTourElementChangedNotification, object: nil, queue: nil) { [weak self] (notification) in
```

Comment:
Do we really need to store that on an `observer` var?

Reply:
Fixed! d83f15a